docusaurus/docs/cloud/projects/collaboration.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Collaboration
+displayed_sidebar: cloudSidebar
+description: Share your projects on Strapi Cloud to collaborate with others.
+canonicalUrl: https://docs.strapi.io/cloud/projects/collaboration.md
+sidebar_position: 1
+---
+
+# Collaboration on projects
+
+Projects are created by a user via his Strapi Cloud account, which is linked to their GitHub account. Strapi Cloud users can share their projects to anyone else, so these new users can have access to the Cloud dashboard and collaborate on that project, without the project owner to ever have to share their GitHub credentials.
```

Comment:
```suggestion
Projects are created by a user via their Strapi Cloud account, which is linked to their GitHub account. Strapi Cloud users can share their projects to anyone else, so these new users can have access to the Cloud dashboard and collaborate on that project, without the project owner to ever have to share their GitHub credentials.
```

I believe you missed to avoid adding gender to the first pronoun, as you did it in the second part of the sentence  🤗